sdk/typescript/src/rpc/websocket-client.ts

```diff
@@ -18,7 +18,7 @@ type NotificationMessageParams = {
 	result: object;
 };
 
-type SubscriptionRequest<T = any> = {
+export type SubscriptionRequest<T = any> = {
```

Comment:
not sure I should have exported this, it may not actually be needed... I'll probably revert this